nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,127 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the Install drive {#sec-installation-booting}
 
-NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
-installation is by and large the same as a BIOS installation. The
-differences are mentioned in the steps that follow.
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Assuming your computer is still running, plug in the install drive
+and restart your computer.
+
+2.   Most computers will allow to select a drive to temporary boot from by
+pressing a special key, usually <kbd>F12</kbd>, but sometimes <kbd>Esc</kbd>
+or another function key. Refer to the screen or your computer's documentation.
+
+3.   Press <kbd>F12</kbd> (or the appropriate key) and select the install drive,
+usually "USB-HDD" or something containing the word "USB", but the wording may vary.
+If your computer support BIOS and UEFI boot, choose the UEFI option.
+If you choose the incorrect drive, your computer will likely continue to boot
+as normal. Just restart your computer and pick a different drive in that menu.
```

Comment:
*Just*?

```suggestion
If you choose the incorrect drive, your computer will likely continue to boot
as normal. In that case restart your computer and pick a different drive in that menu.
```

*Just* is almost always a poor choice when connecting sentences or ideas in docs, IMO. Here it could be understood as passing a judgement that "it's simple, *just* pick the right thing". A more neutral formulation feels better IMO.

Reply:
yes, it can be very frustrating when you do that for the first time and it does not work as you want